docs/Community/governance/README.md

```diff
@@ -0,0 +1,40 @@
+Open Enclave Community Governance Committee
+===========================================
+
+The [charter](charter.md) defines the scope and processes by which the CGC operates.
+
+Meetings: public CGC meetings are held on Wednesdays (bi-weekly) (*link TBD*)
```

Comment:
The meetings are not currently public, only the minutes. We need space to discuss things confidentially. We may want to have a public meeting (I believe this was written as a suggestion in these documents previous iteration) and so I am in support of having regular, public community meeting.

Reply:
Is there anything (other than manually watching who's in the meeting) that enforces that it's public today?  The minutes link below at https://aka.ms/openenclave/cgc contains the link to the Zoom meeting, so the link is currently public if the minutes are public, and the Zoom meeting currently does not require a password or anything.

Reply:
Let me rephrase, we have not invited the public to these meetings. Sure if people start randomly showing up perhaps we need to add a password etc. but regardless, the committee has chosen not to make these meetings public.

Reply:
updated the doc to reflect today's reality (that we don't really have public meetings)